fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/SuggestionPrompt.kt

```diff
@@ -0,0 +1,152 @@
+package com.microsoft.fluentui.tokenized.controls
+
+import androidx.compose.foundation.*
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicText
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.ControlTokens
+import com.microsoft.fluentui.theme.token.FluentIcon
+import com.microsoft.fluentui.theme.token.Icon
+import com.microsoft.fluentui.theme.token.controlTokens.SuggestionPromptInfo
+import com.microsoft.fluentui.theme.token.controlTokens.SuggestionPromptTokens
+
+data class Suggestions(
+    val text: String,
+    val maxLines: Int = Int.MAX_VALUE,
+    val onClick: (() -> Unit)? = null
+)
+
+//Tags for testing
+private const val ACTION_ICON = "Action Icon"
+
+@Composable
+fun SuggestionPrompt(
+    suggestions: MutableList<Suggestions>,
+    modifier: Modifier = Modifier,
+    actionIcon: FluentIcon? = null,
+    suggestionPromptToken: SuggestionPromptTokens? = null
+) {
+    val themeID =
+        FluentTheme.themeID    //Adding This only for recomposition in case of Token Updates. Unused otherwise.
+    val token = suggestionPromptToken
+        ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.SuggestionPrompt] as SuggestionPromptTokens
+    val suggestionPromptInfo = SuggestionPromptInfo()
+
+    Row(
```

Comment:
Use LazyRow for optimal rendering.

Reply:
Can't use LazyRow because we had to use Intrinsics.MaxSize for height. So that if any suggestion has maxLines more than 1, still all suggestions need to maintain that height.